mailpoet/lib/Subscribers/SubscriberSaveController.php

```diff
@@ -229,6 +229,8 @@ public function createOrUpdate(array $data, ?SubscriberEntity $subscriber): Subs
     // wipe any unconfirmed data at this point
     $subscriber->setUnconfirmedData(null);
 
+    $subscriber->setValidationGroups(['Saving', 'Default']);
```

Comment:
Why do we need to pass both `Saving` and `Default` here? Looking at the integration test I would expect just `Saving` should be enough. If that's needed for some reason it might be a good idea to add a comment.

Reply:
If we just pass `Saving`, only the email check will be performed on an entity. But we need to validate everything else (`Default` group) + the email (`Saving` group). I'll add a comment about it. Do you want me to also update the test so the proper usage pattern will be there too?

Reply:
Done.

Reply:
Aha! Thank you for the explanation and for adding the comment. I think it will also be helpful for the other developers reading the code.